utils/python/CIME/hist_utils.py

```diff
@@ -277,7 +277,7 @@ def compare_baseline(case, baseline_dir=None):
 
     for bdir in dirs_to_check:
         if not os.path.isdir(bdir):
-            return False, "ERROR baseline directory '%s' does not exist" % bdir
+            return False, "ERROR BFAIL baseline directory '%s' does not exist" % bdir
```

Comment:
I think this should use the constant defined in test_status.py (currently named TEST_BFAIL_STATUS) rather than hard-coding the string 'BFAIL'.